src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -69,6 +69,8 @@ private static boolean commonHelp(String mode, boolean canConnectToRemote) {
         System.out.println("    --exe <executable for corefile>");
         if (canConnectToRemote) {
             System.out.println("    --connect [<serverid>@]<host>[:registryport][/servername] To connect to a remote debug server (debugd).");
+            System.out.println();
+            System.err.println("WARNING: --connect is deprecated and will be removed in a future release.");
```

Comment:
This is `jhsdb $TOOL --help` output. Shouldn't this just have "(deprecated)" before "To connect ..." rather than a blank line and warning?  I assume the changes should only print "WARNING" if the user attempts to run `jhsdb $TOOL -connect ..`.

Reply:
OK yes understood, we can keep things less ALARMING unless you try and use the options.

--connect is already a long line, so adding "(deprecated)" I think gets lost.  We can do:

```
bash-4.2$ build/linux-x64/images/jdk/bin/jhsdb jstack --help
    --locks                 To print java.util.concurrent locks.
    --mixed                 To print both Java and native frames (mixed mode).
    --pid <pid>             To attach to and operate on the given live process.
    --core <corefile>       To operate on the given core file.
    --exe <executable for corefile>
    --connect [<serverid>@]<host>[:registryport][/servername] To connect to a remote debug server (debugd).

    The --connect is deprecated and will be removed in a future release.
    The --core and --exe options must be set together to give the core
    file, and associated executable, to operate on. They can use
    absolute or relative paths.
    The --pid option can be set to operate on a live process.
    The --connect option can be set to connect to a debug server (debugd).
    --core, --pid, and --connect are mutually exclusive.

    Examples: jhsdb jstack --pid 1234
          or  jhsdb jstack --core ./core.1234 --exe ./myexe
          or  jhsdb jstack --connect serverid@debugserver:1234/servername
bash-4.2$
```


Actually the text "The --connect option can be set to connect to a debug server (debugd)." is in there twice.  Will remove the second one.